demos/background_subtraction_demo/python/README.md

```diff
@@ -0,0 +1,178 @@
+# Background subtraction Python\* Demo
+
+![example](./background_subtraction.gif)
+
+This demo shows how to perform background subtraction using OpenVINO.
+
+> **NOTE**: Only batch size of 1 is supported.
+
+## How It Works
+
+The demo application expects an instance segmentation model in the Intermediate Representation (IR) format with the following constraints:
+1. for instance segmentation models based on `Mask RCNN` approach:
+    * One input: `image` for input image.
+    * At least three outputs including:
+        * `boxes` with absolute bounding box coordinates of the input image and its score
+        * `labels` with object class IDs for all bounding boxes
+        * `masks` with fixed-size segmentation heat maps for all classes of all bounding boxes
+2. for instance segmentation models based on `YOLACT` approach:
```

Comment:
as demo supports yolact-resnet50-fpn-pytorch model and you even mention this in demo description, it makes sense to add this model to models.lst and include to test cases

Reply:
`yolact-resnet50-fpn-pytorch` added to readme, models.lst and test cases.